front/src/applications/editor/tools/routeEdition/components/EditRouteMetadata.tsx

```diff
@@ -12,25 +12,25 @@ import { FiSearch } from 'react-icons/fi';
 import { EditorEntity, WayPointEntity } from 'types/editor';
 import { addFailureNotification, addSuccessNotification } from 'reducers/main';
 
-import { EditRoutePathState, EditRouteMetadataState, RouteEditionState } from '../types';
+import { useOsrdSelectors } from 'common/osrdContext';
+import EditorContext from 'applications/editor/context';
+import { LoaderFill } from 'common/Loader';
+import { getMixedEntities } from 'applications/editor/data/api';
 import {
   getRoutesLineLayerProps,
   getRoutesPointLayerProps,
   getRoutesTextLayerProps,
-} from '../../../../../common/Map/Layers/Routes';
-import colors from '../../../../../common/Map/Consts/colors';
-import { getEditRouteState, getEmptyCreateRouteState, getRouteGeometryByRouteId } from '../utils';
-import EditorContext from '../../../context';
-import { getMixedEntities } from '../../../data/api';
-import { LoaderFill } from '../../../../../common/Loader';
-import { ConfirmModal } from '../../../../../common/BootstrapSNCF/ModalSNCF/ConfirmModal';
-import { save } from '../../../../../reducers/editor';
+} from 'common/Map/Layers/Routes';
+import colors from 'common/Map/Consts/colors';
+import { ConfirmModal } from 'common/BootstrapSNCF/ModalSNCF/ConfirmModal';
+import { save } from 'reducers/editor';
+import { getMapStyle } from 'reducers/map/selectors';
+import { ExtendedEditorContextType } from 'applications/editor/tools/editorContextTypes';
+import EntityError from 'applications/editor//components/EntityError';
+import { NEW_ENTITY_ID } from 'applications/editor//data/utils';
```

Comment:
Thank you for correcting these imports !
```suggestion
import EntityError from 'applications/editor/components/EntityError';
import { NEW_ENTITY_ID } from 'applications/editor/data/utils';
```

Reply:
Done